patterns/hidden-404.php

```diff
@@ -6,16 +6,16 @@
  */
 ?>
 
-<!-- wp:heading {"className":"has-text-align-center","fontSize":"xx-large"} -->
-<h2 class="has-text-align-center has-xx-large-font-size"><?php echo esc_html_x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ); ?></h2>
+<!-- wp:heading {"className":"has-text-align-center"} -->
+<h2 class="has-text-align-center"><?php echo esc_html_x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ); ?></h2>
```

Comment:
I think removing the font size here works well.

Should we make this heading a h1 similar to the other templates? This would make the default font size the same as other templates too.